### ISSUE TITLE
poor test results - > recommendations for improvement

### ISSUE THREAD

**Issue by shxnying** [junyanz/pytorch-CycleGAN-and-pix2pix#1600]
Hi all, I'm relatively new to the field of AI, and I've been working on training a CycleGAN model for a specific task. I've trained the model 200 times using the default settings, but I'm not satisfied with the quality of the results. The generated fake images are unclear and do not meet my expectations. My test results for 145 epochs, 170 epochs and 200 epochs doesn't seem too different

I'm looking for recommendations to improve my CycleGAN model's performance. Can anyone please provide me with some suggestions?

Thank you!!!!

**Comment by JustinasLekavicius** [comment]
Default settings may not be suitable for all tasks, and they would need to be tweaked for your specific task. What is your input data, and expectations for output data? What are the A and B domains? Also, you may need to train for more than 200 epochs for better results.

**Comment by Rjaat** [comment]
Hi @JustinasLekavicius 
Thanks for your input! I’m working with thermal and optical images in .tiff format (around 23k total). Even though the generator loss drops significantly from ~40 to 2-3 within 20 epochs, the generated images remain noisy. If you have any suggestions on how to address this issue, I’d love to hear them!
